src/rev_ai/apiclient.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using rev.ai
+"""
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+import os
+import requests
+import logging
+import json
+from models import (
+    Job,
+    JobSubmitOptions,
+    Account,
+    Transcript
+)
+
+LOG = logging.getLogger(__name__)
```

Comment:
unused, delete?
usage:
`LOG.info("sending job")`
`LOG.critical("server not found")`